<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MechanicalDesign</class>
 <widget class="QMainWindow" name="MechanicalDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面UI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>设计信息</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="font">
           <font>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>设计操作</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="btnDelete">
             <property name="text">
              <string>删除设计</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnDeleteAll">
             <property name="text">
              <string>删除所有设计</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>显示设计</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QRadioButton" name="radioKey">
             <property name="text">
              <string>键的设计</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioGear">
             <property name="text">
              <string>齿轮设计</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioBearing">
             <property name="text">
              <string>轴承设计</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="radioAxle">
             <property name="text">
              <string>轴的设计</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioAll">
             <property name="text">
              <string>所有设计</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="chkSort">
             <property name="text">
              <string>按设计标签排序</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>显示主要设计信息</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPlainTextEdit" name="textEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="actSave"/>
   <addaction name="actOutput"/>
   <addaction name="separator"/>
   <addaction name="actGear"/>
   <addaction name="actBearing"/>
   <addaction name="actKey"/>
   <addaction name="actAxle"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actOpenDB">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/open3.bmp</normaloff>:/images/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>打开数据库</string>
   </property>
   <property name="toolTip">
    <string>开始数据库</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/save1.bmp</normaloff>:/images/images/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>保存设计</string>
   </property>
   <property name="toolTip">
    <string>保存设计到数据库</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actOutput">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/delfold1.bmp</normaloff>:/images/images/delfold1.bmp</iconset>
   </property>
   <property name="text">
    <string>输出设计信息</string>
   </property>
   <property name="toolTip">
    <string>输出设计信息到文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actGear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/gear.jpg</normaloff>:/images/images/gear.jpg</iconset>
   </property>
   <property name="text">
    <string>齿轮设计</string>
   </property>
   <property name="toolTip">
    <string>齿轮设计</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actKey">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/key.jpg</normaloff>:/images/images/key.jpg</iconset>
   </property>
   <property name="text">
    <string>键的设计</string>
   </property>
   <property name="toolTip">
    <string>键的设计</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actAxle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/axle.jpg</normaloff>:/images/images/axle.jpg</iconset>
   </property>
   <property name="text">
    <string>轴的设计</string>
   </property>
   <property name="toolTip">
    <string>轴的设计</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actBearing">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/bearing.jpg</normaloff>:/images/images/bearing.jpg</iconset>
   </property>
   <property name="text">
    <string>轴承设计</string>
   </property>
   <property name="toolTip">
    <string>轴承的设计</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/exit.bmp</normaloff>:/images/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出设计</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MechanicalDesign</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
